src/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+  margin: unset;
+}
+
+body div {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .star:nth-child(-n) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--1 .star:nth-child(-n + 1) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--2 .star:nth-child(-n + 2) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--3 .star:nth-child(-n + 3) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--4 .star:nth-child(-n + 4) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
+
+.stars--5 .star:nth-child(-n + 5) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
```

Comment:
These `.stars--N` classes have the same style, you can group then into one using `,`, check the [styleguide](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)